src/glue/Types.hpp

```diff
@@ -0,0 +1,10 @@
+#ifndef GLUE_TYPES_HPP
+#define GLUE_TYPES_HPP
+
+enum GLUESCRIPTEVENT {
+    SET_GLUE_SCREEN = 0,
+    OPEN_REALM_LIST = 3,
```

Comment:
Pretty sure this is `9`, not `3`.

Reply:
Yes, you're right. Thanks for the feedback.